### PR TITLE
delete cart item

### PR DESCRIPTION
**Pull request by songtv-0006**

### CODE REVIEW

**Comment by duccs-0007** [comment]
OK

**Comment by songtv-0006** [comment]
ready

**Comment by songtv-0006** [comment]
ready

**Comment by songtv-0006** [comment]
ready

**Comment by songtv-0006** [comment]
ready

File: resources/views/carts/index.blade.php [hunk 1]
```diff
@@ -154,11 +154,18 @@
...
         </section>
     @else
-        <div class="row">
-            <div class="col-sm-6 col-md-6 col-md-offset-3 col-md-offset-3">
-                <h2>@lang('content.no_item_in_cart')</h2>
+        <section class="cart_area cart_empty">
+            <div class="container">
+                <div class="cart_inner">
+                    <div class="alert alert-danger alert_font" role="alert">
+                            @lang('content.no_item_in_cart')
```
Comment (tranducquoc) [inline resources/views/carts/index.blade.php:161]: indent

**Pull request merged**